tests/ignite/contrib/metrics/test_fid.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+import pytest
+
+from ignite.contrib.metrics import FID
+
+try:
+    from scipy import linalg
+except ImportError:
+    raise RuntimeError("This contrib test requires scipy to be installed.")
+
+
+def test_fid():
```

Comment:
@kamahori about meaningful tests, I'd like to do the following:
- let's use a real image from internet (<300px) and add it into `tests/ignite/contrib/metrics/assets`
- manually compute FID score for this image and its slightly noised version. Otherwise use one of those referenced implementation. Let's use computed score as true score.
- compute FID score with ignite's implementation and compare scores.